src/domain/youthProfile/youthProfileSchemas.ts

```diff
@@ -89,19 +106,26 @@ const basicInformationSchema = Yup.object().shape({
         }
       ),
       city: Yup.string()
-        .min(2, 'validation.tooShort')
-        .max(255, 'validation.tooLong'),
+        .min(2)
+        .max(255),
     })
   ),
   phone: Yup.string()
-    .min(6, 'validation.phoneMin')
-    .required('validation.required'),
+    .min(6)
+    .required()
+    .label('fields.phoneNumber'),
 });
 
 const additionalInformationSchema = Yup.object().shape({
-  schoolName: Yup.string().max(128, 'validation.tooLong'),
-  schoolClass: Yup.string().max(10, 'validation.tooLong'),
-  photoUsageApproved: Yup.string().required('validation.required'),
+  schoolName: Yup.string()
+    .max(128)
+    .label('fields.schoolName'),
+  schoolClass: Yup.string()
+    .max(10)
+    .label('fields.schoolClass'),
```

Comment:
There shouldn't be any scenario where these fields are required. But I guess it won't hurt to leave these here.

Reply:
Did you comment the correct lines? AFAIK these two fields are not required.

Reply:
Exactly, maybe I miss understood something, but aren't these labels for displaying text "Required + field name"?

Reply:
Ah, now I understand. The labels are also used for `min` and `max` errors. See here:

https://github.com/City-of-Helsinki/youth-membership-ui/pull/208/files#diff-85a7f9f12f8ccede1618ba1ef3c66c7b8cbb6da984cd2ef77af1400672330e31R215-R216